### ISSUE TITLE
Exists a way to make a right-click on a clickable element?

### ISSUE THREAD

**Issue by Farmijo** [mmarquee/ui-automation#65]
Great framework, is helping me a lot.

In need to right-click elements that are clickable. Exists a way currently, or there is an intention to do it?

**Comment by mmarquee** [comment]
The underlying automayion library that is in Windows 8.1 and beyond (i.e. NOT Windows 7) has a method called `showContextMenu`. This is exposed in this library in the `AutomationElement.showContextMenu` method. However, it seems to be very dependent on the way in which context menus are implemented in the actual application, so it will fail sometime.s to actually display the context menu.

**Comment by Farmijo** [comment]
Yep, does seems to work for the app which i'm applying that. If i could get the coords for the UIAutomation elements declared, at least I could send a right click with robot.
I will try back to work. 
Thanks :)

**Comment by Farmijo** [comment]
Does not*
